fineract-core/src/main/java/org/apache/fineract/portfolio/savings/data/SavingsAccountSummaryData.java

```diff
@@ -56,13 +56,15 @@ public class SavingsAccountSummaryData implements Serializable {
     private LocalDate interestPostedTillDate;
     private LocalDate prevInterestPostedTillDate;
     private transient BigDecimal runningBalanceOnInterestPostingTillDate = BigDecimal.ZERO;
+    private LocalDate accruedTillDate;
+    private BigDecimal totalInterestAccrued;
```

Comment:
Dont we need accrued fee and penalty?

Reply:
_The Accrual Fee mainly will be in other PR_

Reply:
I am wondering whether it would be better to send support for fee and penalty as well in the same PR or disable the job till it is not complete. THoughts?